docs/content/usage/mailinglists.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Mailinglists
+description: Using the Mailinglists API
+status: Stable
+---
+
+import {CheckIcon} from '@primer/styled-octicons';
+
+## Introduction
+
+The Mailinglists API allows you to interact with our Mailinglists data.
+
+## Usage
+
+### Create
+
+Create a Mailinglist
+
+**Parameters**
+
+| Property      | Type      | Required                      | Example                                   |
+| :--           | :--       | :-:                           | :--                                       |
+| business_id   | Xuid      | <CheckIcon color="green.5"/>  | 'a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e'    |
+| name          | String    | <CheckIcon color="green.5"/>  | 'My List'                                 |
+| description   | String    | <CheckIcon color="green.5"/>  | 'My new shiny Mailinglist'                |
+
+
+**Usage**
+
+```javascript
+    await Xedi.Mailinglist.create({
+        "businees_id": "a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e",
+        "name": "My Mailinglist",
+        "description": "My Shiny new Mailinglist",
+    })
+```
+
+### List
+
+List All Mailinglists for a business
+
+**Parameters**
+
+| Property      | Type      | Required                      | Example                                   |
+| :--           | :--       | :-:                           | :--                                       |
+| businessXuid  | Xuid      | <CheckIcon color="green.5"/>  | 'a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e'    |
+
+
+**Usage**
+
+```javascript
+    await Xedi.Mailinglist.list("a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e");
+```
```

Comment:
I perhaps wouldn't call this list as the intranet will need a list endpoint that is an actual index. getByBusiness would be more appropriate